pkg/reconciler/revision/table_test.go

```diff
@@ -623,7 +632,7 @@ func changeContainers(deploy *appsv1.Deployment) *appsv1.Deployment {
 	return deploy
 }
 
-func rev(namespace, name string, ro ...RevisionOption) *v1.Revision {
+func rev(namespace, name, containerName string, ro ...RevisionOption) *v1.Revision {
```

Comment:
Would the changes be simpler if we default the `containerName` to be the same as `name`, and users can override this with `RevisionOption`?